corpus/events/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path("", views.dashboard, name="dashboard"),
+    path("create_event", views.create_event, name="create_event"),
```

Comment:
Maybe make the path `new`. Since the app is already routed to `/events`, we don't need `event` in the path names again.